calico/networking/configuring/bgp-to-workload.mdx

```diff
@@ -0,0 +1,302 @@
+- [BGPFilter](../../reference/resources/bgpfilter.mdx) - Defines a BGP filter
+  to control which routes are imported or exported between BGP peers.
+- [BGPConfiguration](../../reference/resources/bgpconfig.mdx) - Configures global
+  BGP settings for the cluster.
+
+The [BGP Guide](./bgp.mdx) provides a general overview of these concepts.
+
+## Supported BGP topologies
+
+Both parent and nested clusters must be configured to use non-overlay BGP
+networking.
+
+eBGP must be used both for peering between parent cluster and top-of-rack
+router and for peering between parent cluster and nested workload cluster.
+Each of the ToR, parent cluster, and nested cluster must have a unique AS number.
+
+iBGP peerings to workloads or ToR routers are not supported:
+
+- iBGP to the ToR router is not supported because there is currently no way to
+  configure "next hop self" behaviour on that peering (so the ToR would receive
+  the nested cluster's routes with the workload's IP as next hop, rather than
+  the parent cluster node's IP).
+
+- iBGP to the nested cluster has not been tested. It may work with appropriate
+  `BGPFilter`s to ensure the correct routes are re-advertised.
+
+Peering with workloads and using a BGP mesh within the cluster is not supported;
+this combination has not been tested and it is likely to require additional
+`BGPFilter`s to ensure routes are correctly re-advertised.
+
+## Supported KubeVirt networking modes
+
+Only KubeVirt's "bridged" networking mode is supported. This is because the
+peering from parent cluster to the workload is done using the "pod IP" of the
+workload.  This must agree with the IP that the workload itself uses to
+source traffic.
+
+## Before you begin...
+
+Ensure you can access `projectcalico.org/v3` resources, either by installing the
+$[prodname] [API server](../../operations/install-apiserver.mdx) (which makes
+them available through `kubectl`), or by using [calicoctl](../../operations/calicoctl/install.mdx).
+
+## How to
+
+- [Configure global BGP settings](#configure-global-bgp-settings)
+- [Create BGPFilter resources](#configure-bgpfilter-resources)
+- [Configure parent cluster BGP topology](#configure-parent-cluster-bgp-topology)
+- [Configure BGP peering with workloads](#configure-bgp-peering-with-workloads)
+- [Configure more than one nested cluster](#configure-more-than-one-nested-cluster)
+
+### Configure global BGP settings
+
+In order to ease configuration of the `BGPPeer` resources in the nested clusters,
+the workload BGP peering feature uses a single virtual IP (per IP version) that
+is provisioned on every node in the parent cluster. The workloads are configured
+to peer with this virtual IP rather than needing to know the specific IP of their
+parent node.
+
+To avoid conflicts with your network, the addresses are configurable through
+the `BGPConfiguration` resource and there are no defaults. They must be
+configured to enable the feature:
```

Comment:
I'm not entirely sure that a list is useful here. Consider formatting as paragraphs.